axelnguyen0701/book-exchange
# Chat frontend michael

**Easiest way to test**

You can just run the front end only with npm run dev to test

1. Create two brand new accounts on meta mask (can be done on mainnet for simplicity sake)
2. Sign in with the account you want to receive messages. The accounts DID will be shown on the homepage, you will receive an error because the account is not connected to the network (irrelevant to this and can be closed)
3. In message.js line 87, replace the to with the DID of your account.
4. Sign in with the second account, go to search, press search, view listings for forensic psychology, press the message button the listing and send a message.
5. Check the messages tab to see if message sent on the senders side
6. Log into the recipient account and check if the message delivered. You can reply as well with either the enter key or pressing the send icon. State changes update the conversation, so pressing a key in the text box will have the new sent/received messages show up. (to be fixed)
